src/Bicep.Cli/Arguments/FormatArguments.cs

```diff
@@ -45,19 +46,38 @@ public FormatArguments(string[] args, IOContext io) : base(Constants.Command.For
                         break;
 
                     case "--newline":
```

Comment:
Lots of common code here, consider extracting.

Reply:
Yeah, makes sense. I'd prefer to send a dedicated PR to do the refactoring since the same pattern is repeated in the other arguments classes as well.

Reply:
Actually, I think it might be beneficial to hold off on the refactoring for now since we've been planning a migration to `System.CommandLine`.